scripts/lib/mkcloud-libvirt.sh

```diff
@@ -27,25 +27,27 @@ function workaround_bsc928384()
 # Returns success if the config was changed
 function libvirt_configure_libvirtd()
 {
-    chkconfig libvirtd on
+    systemctl enable libvirtd
 
     local changed=
 
     # needed for HA/STONITH via libvirtd:
     confset /etc/libvirt/libvirtd.conf listen_tcp 1            && changed=y
     confset /etc/libvirt/libvirtd.conf listen_addr '"0.0.0.0"' && changed=y
     confset /etc/libvirt/libvirtd.conf auth_tcp '"none"'       && changed=y
-    workaround_bsc928384 && changed=y
+    if is_suse ; then
+        workaround_bsc928384 && changed=y
```

Comment:
This bug+workaround is not SUSE-specific, eventhough we tracked it in SUSE's bugzilla